Modules/Core/Common/include/itkThreadSupport.h

```diff
@@ -62,7 +62,7 @@ using ITK_THREAD_RETURN_FUNCTION_CALL_CONVENTION = itk::ITK_THREAD_RETURN_TYPE;
 constexpr size_t ITK_MAX_THREADS = ITK_DEFAULT_MAX_THREADS;
 using MutexType = HANDLE;
 using FastMutexType = CRITICAL_SECTION;
-using ThreadFunctionType = unsigned int(__stdcall *)(void *);
```

Comment:
This change might be a problem.

Reply:
Yes, it is a problem: `D:\a\1\s\Modules\Core\Common\include\itkThreadSupport.h(65): error C2059: syntax error: 'static_cast'`.

Reply:
Thank you.  I will fix that.